api/admin/service.md

```diff
@@ -110,40 +96,37 @@ curl -X POST --data '{
 }' -H 'content-type:application/json;' 127.0.0.1:9650/ext/admin
 ```
 
-**Example Response:**
+**Example Response**:
 
-```json
+```
 {
   "jsonrpc": "2.0",
   "id": 1,
   "result": {}
 }
 ```
 
-Now, instead of interacting with the blockchain whose ID is
-`sV6o671RtkGBcno1FiaDbVcFv2sG5aVXMZYzKdP4VQAWmJQnM` by making API calls to
-`/ext/bc/sV6o671RtkGBcno1FiaDbVcFv2sG5aVXMZYzKdP4VQAWmJQnM`, one can also make calls to
-`ext/bc/myBlockchainAlias`.
+Now, instead of interacting with the blockchain whose ID is `sV6o671RtkGBcno1FiaDbVcFv2sG5aVXMZYzKdP4VQAWmJQnM` by making API calls to `/ext/bc/sV6o671RtkGBcno1FiaDbVcFv2sG5aVXMZYzKdP4VQAWmJQnM`, one can also make calls to `ext/bc/myBlockchainAlias`.
 
-### `admin.getChainAliases`
+### admin.getChainAliases
```

Comment:
what were the `` removed here but not for `admin.alias`, `admin.aliasChain`, etc.?